RELEASE/scripts/sl_ascend/sl_batpath.ash

```diff
@@ -355,6 +355,51 @@ boolean bat_shouldEnsorcel(monster m)
 	return false;
 }
 
+boolean bat_multicraft(string mode, boolean [item] options)
+{
+	if(my_class() != $class[Vampyre])
+		return false;
+	if(item_amount($item[blood bag]) == 0)
+		return false;
+
+	foreach ingredient in options
+	{
+		if(craft(mode, 1, $item[blood bag], ingredient) > 0)
```

Comment:
This is the reason that you're getting those error messages. You need an `item_amount` check on `ingredient` before the craft call to avoid it.